smoke-test/src/test/java/org/opennms/smoketest/TopologyIT.java

```diff
@@ -70,10 +72,9 @@
  *
  * @author jwhite
  */
+@Slf4j
```

Comment:
Logger construction is here replaced with an annotation (small improvement). Read more: https://projectlombok.org/features/log